config/environments/test.rb

```diff
@@ -45,8 +45,6 @@
   # Raises error for missing translations.
   # config.i18n.raise_on_missing_translations = true
 
-  config.active_job.queue_adapter = :inline
```

Comment:
Looks like this was set deliberately in b7ae780e58dbcd4fcd5ca56b00c790c155380c1f - we should have a think about whether to leave this alone.